exchanges/assets.py

```diff
@@ -0,0 +1,49 @@
+class Asset(object):
+
+    def __init__(self, name, atype, iso4217_a3=None, abbreviation=None):
```

Comment:
Since many of our tools expect asset pairs, it could be nice to have syntactic sugar to define them. For example, we could overload `__add__` to return a tuple. Then you could write
```
print(kraken.lastest_trade(Bitcoin + Dollar))
```